.github/workflows/release.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Build and publish Docker image with authentication
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
-          name: farao/gridcapa-app
+          name: farao/gridcapa-app-auth
           username: farao
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-          tags: "v${{ github.event.inputs.releaseVersion }}-auth"
+          tags: "v${{ github.event.inputs.releaseVersion }}, v${{ github.event.inputs.releaseVersion }}-auth"
```

Comment:
Why keeping -auth tag ? perhaps in the same idea we should also get rid of the -no-auth tag ? To be discussed